pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -777,10 +777,17 @@ async def _sync_images(self):
 
         records = record_log.get('build', [])
         operator_nvrs = []
-
+        operator_nvrs_map = {}
+        operator_deps = []
         for record in records:
             if record['has_olm_bundle'] == '1' and record['status'] == '0' and record.get('nvrs', None):
                 operator_nvrs.append(record['nvrs'].split(',')[0])
+            if record['status'] != '0' and record.get('nvrs', None) and record.get('dependents', None):
+                operator_deps.extend([dep for dep in record['dependents'].split(',') if dep not in operator_deps])
+            operator_nvrs_map[record['distgit']] = record.get('nvrs')
```

Comment:
why is this called operator_nvrs_map, since it is storing every image's (operator or not) distgit -> nvrs ? 
Something like `distgit_to_nvr_map` makes it obvious ?